hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/offlineImageViewer/PBImageDelimitedTextWriter.java

```diff
@@ -146,7 +146,13 @@ public String build() {
   PBImageDelimitedTextWriter(PrintStream out, String delimiter,
                              String tempPath, boolean printStoragePolicy)
       throws IOException {
-    super(out, delimiter, tempPath);
+    this(out, delimiter, tempPath, printStoragePolicy, 1, "-");
```

Comment:
I am confused why we use static string "-" rather than null. any other consideration?

Reply:
Considering the default value of `outputFile` in `OfflineImageViewerPB.java` is "-", I continue this style.
```java
    String outputFile = cmd.getOptionValue("o", "-");

```